frontend/public/locales/en/communication-preference.json

```diff
@@ -0,0 +1,14 @@
+{
+  "page-title": "Communication preferences",
+  "breadcrumbs": {
+    "preferred-communication": "Communication preferences"
+  },
+  "id": "ID",
```

Comment:
Did you intend on using this i18n key anywhere? It's currently not being referenced. 

(Same comment for French)

Reply:
removed